File: CopyArrayinReverse.java
import java.util.Scanner;

public class CopyArrayinReverse {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter size: ");
        int n = sc.nextInt();
        int[] arr = new int[n];
        int[] rev = new int[n];

        System.out.println("Enter elements:");
        for(int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
        }

        for(int i = 0; i < n; i++) {
            rev[i] = arr[n - 1 - i];
        }

        System.out.print("Reversed Array: ");
        for(int num : rev) {
            System.out.print(num + " ");
        }
    }
}

File: python/CopyArrayinReverse.py
# Copy array in reverse order
n = int(input("Enter size: "))
arr = []
print("Enter elements:")
for _ in range(n):
    arr.append(int(input()))
rev = arr[::-1]
print("Reversed Array:", end=" ")
for num in rev:
    print(num, end=" ")
print()
